src/derivative_operators/abstract_operator_functions.jl

```diff
@@ -184,6 +191,25 @@ function Base.convert(::Type{Array}, A::AbstractDerivativeOperator{T}, N::Int=A.
 end
 
 
+function Base.convert(::Type{Array}, A::InteriorOperator{T}, N::Int=A.dimension) where T
```

Comment:
In the interior, you'll want to grab the contiguous `.x` portion out of the lazy array so that way you can avoid the slower indexing. It might be a good idea to make and test a lazy array first.